src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,16 @@
+// Add shared components, directives, and pipes to this module.
+// Do not add global services to this module, add to CoreModule instead.
+
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  imports: [
+    CommonModule,
+  ],
+  declarations: [],
+  exports: [
+    CommonModule,
```

Comment:
We don't, but this is good refactoring step. Re-exporting from the shared module means that we don't have to import it separately in every module that we import the shared module into (which is likely to be all/most of them)